packages/fuel-gauge/src/funding-transaction.test.ts

```diff
@@ -511,7 +511,7 @@ describe('Funding Transactions', () => {
       () => fundedWallet.transfer(receiver.address, transferAmount, provider.getBaseAssetId()),
       new FuelError(
         FuelError.CODES.INVALID_REQUEST,
-        'Transaction is not inserted. Hash is already known'
+        'Transaction input validation failed: Transaction id already exists (id: bacb0b8395b11e8d9a802f38674c5e41bd065567281d8c2f8f791e7760643e6a)'
```

Comment:
```suggestion
    }).rejects.toThrowError(/Transaction input validation failed: UTXO \(id: .*\) does not exist/);
```

Relates to changes [here](https://github.com/FuelLabs/fuel-core/pull/2216/files#diff-763e9a79947aa33db6000118408d6f1d8e639be46a2d6ff54939f097e7986aedR28) and [here](https://github.com/FuelLabs/fuel-core/blob/94fe64e876bbb651311f1e05a6ad77c54366ee65/crates/services/txpool_v2/src/error.rs#L130-L131)